pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = [
     "h5py",
     "softioc>=4.3.0",
     "pandablocks>=0.3.1",
-    "pvi[cli]>=0.4",
+    "pvi[cli]<0.5",
```

Comment:
pvi requires 3.10 for v0.5, it's neccessary to pin this for consistency on `.bob` files between 3.9 and 3.10/3.11.

Maybe we should consider requiring 3.10 for pandablocks-ioc @AlexanderWells-diamond?

Reply:
https://github.com/PandABlocks/PandABlocks-ioc/pull/26 this will require `pvi>=0.5`

Reply:
I don't know why PVI updated itself to Python 3.10 - it appears to be [this PR](https://github.com/epics-containers/pvi/pull/34) that did it, but there are no functional changes to the Python code that warrant dropping lower Python versions - the only changes appear to be in the `typing` module, which will not affect runtime behaviour.

Ultimately I don't really mind what version requirements we have. I'll defer to whatever @coretl and @GDYendell want to do.